infrastructure/kube/templates/relay-maintainer/initcontainer/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "@keep-network/tbtc": ">0.14.0-pre <0.14.0-rc",
+    "@truffle/hdwallet-provider": "^1.0.25",
+    "web3": "1.0.0-beta.55"
```

Comment:
Oof. Probably we can hold here, but we should really bump this to a >1.0 version of web3.

Reply:
Jah